base/expr.jl

```diff
@@ -118,6 +118,17 @@ macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
 
+"""
+    @inline
+
+Small functions typically don't even need the @inline annotation, it does it on its own. By using @inline on bigger functions, an extra nudge can be given to the compiler, like in the example below.
+
+    @inline function bigfunction(x)
```

Comment:
Same as above with using triple backticks for this code block to highlight it correctly.